Resources/Computing/ARCComputingElement.py

```diff
@@ -423,6 +423,13 @@ def getJobOutput( self, jobID, localDir = None ):
 
     isItOkay = job.Retrieve( self.usercfg, arc.URL( workingDirectory ), False )
     if ( isItOkay ):
+      try:
+        with open(outFileName, 'r') as outFile:
+          output = outFile.read()
```

Comment:
A bit dirty, as the outFileName is now read twice ... once here and again in line 433.

To get out of this, I have to define the variable "output" previously. How should I initialise it? Will doing something like

output = ""

be enough between lines 426 and 427?